extensions/acc-core.md

```diff
@@ -197,65 +195,64 @@ description.
 
 ### Registering the account "kaniini" with e-mail address kaniini@example.com:
 
-    C: REG CREATE kaniini mailto:kaniini@example.com passphrase :testpassphrase123
+    C: ACC REGISTER kaniini mailto:kaniini@example.com passphrase :testpassphrase123
     S: 920 kaniini kaniini :Account registered
     S: 927 kaniini kaniini kaniini@example.com :A verification code was sent
-    C: REG VERIFY kaniini 3qw4tq4te4gf34
+    C: ACC VERIFY kaniini 3qw4tq4te4gf34
     S: 923 kaniini kaniini :Account verification successful
     S: 903 kaniini :Authentication successful
 
 ### Registering with the client's current nickname "dan-":
 
-    C: REG CREATE * mailto:dan@example.com passphrase :testpassphrase123
+    C: ACC REGISTER * mailto:dan@example.com passphrase :testpassphrase123
     S: 920 dan- dan- :Account registered
     S: 927 dan- dan- dan@example.com :A verification code was sent
     ...
 
 ### Registering the account "rabbit" with SMS number +11234567890:
 
-    C: REG CREATE rabbit sms:+11234567890 passphrase :testpassphrase123
+    C: ACC REGISTER rabbit sms:+11234567890 passphrase :testpassphrase123
     S: 920 kaniini rabbit :Account registered
     S: 927 kaniini rabbit +11234567890 :A verification code was sent
-    C: REG VERIFY rabbit 123456
+    C: ACC VERIFY rabbit 123456
     S: 923 kaniini rabbit :Account verification successful
     S: 903 kaniini :Authentication successful
 
 ### Registering the account "rabbit" with an unsupported verification method:
 
-    C: REG CREATE rabbit 1vBjNBdhjWFFbbbbVBHJEWBHJWcfbbvjkhbea passphrase :testpassphrase123
+    C: ACC REGISTER rabbit 1vBjNBdhjWFFbbbbVBHJEWBHJWcfbbvjkhbea passphrase :testpassphrase123
     S: 929 kaniini rabbit 1vBjNBdhjWFFbbbbVBHJEWBHJWcfbbvjkhbea :Callback token is not valid
 
 ### Registering the account "rabbit", but providing an invalid verification token:
```

Comment:
Add token for oper overrides (account verified and user notified account was verified for them).